packages/editor/src/client/blocks/teaserGridBlock.tsx

```diff
@@ -131,7 +131,7 @@ export function TeaserGridBlock({value, onChange}: BlockProps<TeaserGridBlockVal
 export interface TeaserBlockProps {
   teaser: Teaser | null
   showGrabCursor: boolean
-  numColumns: number
+  numColumns?: number
```

Comment:
why did you change in that file? I think not related?

Reply:
@bishoymelek I wanted to be able to use the existing teaser functions. Essentially the same thing besides the columns.

Reply:
@penk85 but isn't that affecting other places where this interface already being used?